### ISSUE TITLE
Cannot add extra fields to a nested property

### ISSUE THREAD

**Issue by shusson** [xrobert35/es-mapping-ts#4]
Example:

```typescript
    @EsField({
        type: "nested",
        fieldClass: FormEntity,
        dynamic: "strict"
    })
    form: FormEntity[];
```

`dynamic` will not be added to the mapping.

**Comment by xrobert35** [comment]
Hello shusson, 
Thanks for reporting this, I will correct this asap

**Comment by xrobert35** [comment]
Fixed in 0.0.14
